### DISCUSSION TITLE
Make custom fields in "Projects" possible to be required

### DISCUSSION THREAD

**Discussion by smileBeda** [community/community#70002] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

Project Custom Fields can still not be made required, and this is an often required feature for any project management (thinking of "due date" field in an issue, for example)

Thus when someone opens an issue the field is not required automatically :(

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by justyn** [comment]
This is a current frustration - people create issues in a Project without adding a required field.

It's a big waste of time because they would do it easily if it was not possible to create the issue without it - they just forget.

Please add a "required" property to custom fields!
